### ISSUE TITLE
Hashtag in ATIS labels

### ISSUE THREAD

**Issue by mebrunet** [jianguoz/Few-Shot-Intent-Detection#6]
I'm unfamiliar with the original dataset, but it seems to me like there is an issue with the label files in `Datasets/ATIS/*/label`.  Some labels are the concatenation of two labels with a hashtag. Is this indicating a multiclass classification? If not, the validation set contains a label that the training set does not (`atis_airfare#atis_flight_time`). See terminal output:

```bash
for ds in *; do echo $ds; cat $ds/label | grep "#"; done
test
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_airfare#atis_flight
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airline
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight_no#atis_airline
atis_flight#atis_airfare
atis_flight#atis_airfare
train
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_aircraft#atis_flight#atis_flight_no
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_airline#atis_flight_no
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_flight#atis_airfare
atis_airline#atis_flight_no
atis_ground_service#atis_ground_fare
atis_flight#atis_airfare
valid
atis_flight#atis_airfare
atis_airfare#atis_flight_time
atis_flight#atis_airfare
```

**Comment by jianguoz** [comment]
Hi @mebrunet, thanks for pointing out the potential issues. I am investigating and will give an update soon.

**Comment by jianguoz** [comment]
Hi @mebrunet, sorry for the late reply! Based on my initial investigation. I found that some utterances may have multiple intents. For example, `atis_flight#atis_airfare` means that the utterance has two intents: `atis_flight` and `atis_airfare`.  [Here](https://www.kaggle.com/code/siddhadev/atis-dataset-from-ms-cntk/input?scriptVersionId=10371998) are the intents of the original ATIS dataset.
